sdk/python/kfp_tekton/compiler/compiler.py

```diff
@@ -686,7 +686,7 @@ def compile(self,
               type_check=True,
               pipeline_conf: dsl.PipelineConf = None,
               allow_telemetry: bool = True,
-              enable_artifacts=False):
+              enable_artifacts=True):
```

Comment:
we should flip this and call it `disable_artifacts` as well to be consistent

Reply:
I'm following the same design as kfp here, their `type_check` and `allow_telemetry` is default to true, but in command line they are using `disable-type-check` and `disable-telemetry`

https://github.com/kubeflow/kfp-tekton/blob/master/sdk/python/kfp_tekton/compiler/main.py#L44-L49

Reply:
Okay, let's go with KFP precedence then